### PR TITLE
Metadata in components indexer

### PR DESCRIPTION
**Pull request by milenakrawczyk**
This PR adds metadata to the components indexer.

The SQL schema has been changed due to the old version not properly generating.

### CODE REVIEW

File: indexers/components/components.js [hunk 1]
```diff
@@ -446,6 +448,13 @@ async function getBlock(block: Block) {
             if (result.removed) linesRemoved += result.count;
           });
 
+          let forkSource = metadata?.fork_of ? metadata?.fork_of.split('@')[0]: null;
+          let forkBlockHeight = metadata?.fork_of ? metadata?.fork_of.split('@')[1]: null;
+          if (forkSource === `${componentAuthorId}/widget/${componentName}`) {
```
Comment (milenakrawczyk) [inline indexers/components/components.js:453]: Checking if the fork source is the same as the widget path. If so, the widget was not forked.

File: indexers/components/components.sql [hunk 2]
```diff
@@ -1,14 +1,22 @@
 CREATE TABLE
   "versions" (
-    "id" SERIAL PRIMARY KEY,
-    "block_height" INT NOT NULL,
+    "id" SERIAL,
+    "block_height" BIGINT NOT NULL,
```
Comment (gabehamilton) [inline indexers/components/components.sql:4]: why the change to bigint?

**Review by gabehamilton — APPROVED** [review]

File: indexers/components/components.sql (hunk 2 above)
Comment (milenakrawczyk) [reply indexers/components/components.sql:4]: This change is optional as it will take a lot of time before we hit the max of INT. However, I figured we may do that now since the schema changes anyway which will result in a new indexer. I may revert it if we stick to INT values only.
